### PR TITLE
Experimental

### PR DESCRIPTION
**Pull request by ivanB1975**
Cleaned the code and moved the collection name on the core. In this way the mongodb library is as generic as possible.

### CODE REVIEW

File: src/cf_sample_app_clojure/mongodb.clj [hunk 1]
```diff
@@ -5,25 +5,24 @@
...
             [com.mongodb DB WriteConcern]))
 
-(defn saveToMongo [document]
- (try 
+(defn- connectToMongo []
   (let [credentials (getCredentials "mongodb")
-        uri (get credentials :uri "mongodb://127.0.0.1:27017/monger-test")
-        {:keys [conn db]} (mg/connect-via-uri uri)
-        coll "todos"]
+        uri (get credentials :uri "mongodb://127.0.0.1:27017/monger-test")]
```
Comment (cloudlena) [inline src/cf_sample_app_clojure/mongodb.clj:10]: can you please check the indentation for this file?

Comment (ivanB1975) [inline src/cf_sample_app_clojure/mongodb.clj:10]: You are right it is not correct.

**Comment by ivanB1975** [comment]
Corrected now

File: src/cf_sample_app_clojure/mongodb.clj [hunk 2]
```diff
@@ -5,25 +5,24 @@
...
-    (println "Inserting document")
-    (mc/insert-and-return db coll {:todo document})
-    (println "Document inserted")
-    (println "Closing database")
-    (mg/disconnect conn))
+        uri (get credentials :uri "mongodb://127.0.0.1:27017/monger-test")]
+    (mg/connect-via-uri uri)))
+
+(defn saveToMongo [document coll]
+ (try
```
Comment (cloudlena) [inline src/cf_sample_app_clojure/mongodb.clj]: shouldn't this be indented by 2 spaces?

**Comment by ivanB1975** [comment]
now there are some conflicts being the advanced changed under the experimental. It doesn't matter since we will merge the experimental on the advanced.

**Pull request merged**